main.go

```diff
@@ -192,6 +192,7 @@ func main() {
 	// Set up the storage service.
 	var storagesvc storage.Service
 	{
+		cfg.Storage.TaskQueue = cfg.Temporal.TaskQueue
```

Comment:
What would be the motivation of doing this?

Reply:
i have no idea why i did this it does not seem to cause any bugs when i remove it.